File: codeforces/cf1526A_mean_inequality.hs
-- uninhm
-- https://codeforces.com/contest/1526/problem/A
-- greedy

import Data.List (transpose, sort)

odds (x:xs) = x : evens xs
odds [] = []

evens (_:xs) = odds xs
evens [] = []

split a = [take l a, drop l a]
  where l = length a `div` 2

solve :: [Int] -> [Int]
solve = concat . transpose . split . sort

solveLine =
  unwords . map show . solve . map read . words

main = interact $ unlines . map solveLine . evens . tail . lines

File: codeforces/cf1526A_mean_inequality.py
# uninhm
# https://codeforces.com/contest/1526/problem/A
# greedy

t = int(input())

for _ in range(t):
    n = int(input())
    a = list(map(int, input().split()))
    a.sort()
    b = []
    for i in range(n):
        b.append(a[i])
        b.append(a[i+n])
    print(*b)
